sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobApiTests.java

```diff
@@ -1966,4 +1969,138 @@ public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineN
     public static boolean hasServiceVersion() {
         return ENV.getServiceVersion() != null;
     }
+
+    private static Stream<Arguments> fourMBUploadsV2Supplier() {
+        return Stream.of(
+            Arguments.of(4194305),
+            Arguments.of(4194304),
+            Arguments.of(4194277),
+            Arguments.of(33554432)
+        );
+    }
```

Comment:
Instead of creating new cases, could we add onto existing cases and leverage existing tests? For the sync client, it looks like we could add these cases to `downloadFile` test and the 16MB case to the `uploadAndDownloadToFileDifferentRegionLength` tests.

For the async tests, I'm not sure if I saw any async client tests so if there are not any we can use the cases that we have but instead it may make sense to use the parameterized cases from the synchronous versions